src/main/markdown/articles/superdevmode.md

```diff
@@ -84,13 +84,15 @@ command line arguments. For 2.7, here is the output:
       -[no]precompile               Precompile modules. (defaults to ON)
       -port                         The port where the code server will run.
       -src                          A directory containing GWT source to be prepended to the classpath for compiling.
-      -X[no]enforceStrictResources  EXPERIMENTAL: Don't implicitly depend on "client" and "public" when a module doesn't define any dependencies. (defaults to OFF)
       -workDir                      The root of the directory tree where the code server willwrite compiler output. If not supplied, a temporary directorywill be used.
       -launcherDir                  An output directory where files for launching Super Dev Mode will be written. (Optional.)
+      -style                        Script output style: DETAILED, OBFUSCATED or PRETTY
+      -setProperty                  Set the values of a property in the form of propertyName=value1[,value2...].
       -[no]incremental              Compiles faster by reusing data from the previous compile. (defaults to ON)
-      -sourceLevel                  Specifies Java source level (defaults to auto:1.7)
+      -sourceLevel                  Specifies Java source level (defaults to auto:1.8)
       -logLevel                     The level of logging detail: ERROR, WARN, INFO, TRACE, DEBUG, SPAM or ALL (defaults to INFO)
-      -XjsInteropMode               EXPERIMENTAL: Specifies JsInterop mode: NONE, JS or CLOSURE (defaults to NONE)
+      -XjsInteropMode               EXPERIMENTAL: DEPRECATED: Specifies JsInterop mode: JS or JS_RC
```

Comment:
AFAIK `XjsInteropMode` will be gone by the 2.8 release so it might be a bit too early to update this doc; or maybe say it's 2.8.0-beta1 and make sure to revise it again when 2.8 "final" is here?